### DISCUSSION TITLE
Welcome to the Siege Engine forum!

### DISCUSSION THREAD

**Discussion by jonjondev** [CapsCollective/siege-engine#14] [Announcements]
## Welcome to the Siege Engine forum!
The Siege Engine team encourages the use of GitHub Discussions as a place to interact with other members of our community outside of issues.

### Here you can:
* Ask questions about the project or the technology we use
* Get help with any issues you're having, Siege-specific or otherwise
* Soundboard any cool ideas you have with the rest of the community before going to write up a feature request

Keep in mind that this project adheres to the [Contributor Covenant code of conduct](https://github.com/CapsCollective/siege-engine/blob/feature/docs-update/CODE_OF_CONDUCT.md), and by participating in this forum, you are expected to uphold this code at all times.

We welcome anyone who wishes to introduce themselves to comment below! We're always keen to get to know the people who participate in open source projects.

**Comment by jonjondev** [comment]
_Guess I'll go first_

Hi all! I'm Jonathan and I'm one of the co-creators and maintainers of Siege Engine. I started working on this project with @Raelr about two years ago after having a real hard time getting to grips with building games from scratch, and without reiterating the project's README, I'll say that the weight and opacity of commercial or general game engines was sometimes bothersome. At the time, we dreamt up a future where we could know what our game was doing top-to-bottom, and change is as we needed.

Since then we've been working to build up and tear down code, making mistakes, and trying new things to see what works for us. It's been a ton of fun and a lot of good learning, but we always wanted to keep it understandable to new collaborators and share what we've learned in the process.

Looking forward to seeing where this can go, and who joins us in that journey along the way!

**Comment by Raelr** [comment]
Hello! I'm Aryeh and I'm another one of the co-creators! I'm a bit of a geek - I have an unhealthy obsession with opening things to see what's inside and seeing if I can replicate them. I have personally been on a journey build a game engine for almost three years, having spent a lot of time trying to understand what goes behind these types of tools. It's been a long journey and I still feel like I've only seen the tip of the ice-berg. 

I love this type of work, and I hope that mine and @J-Mo63's passion for this project will be reflected in the craftsmanship and design of Siege. I'm so excited to have a chance to share this with others and see just how far this will go!
